src/python/analyzer_executor/src/analyzer_executor_lib/analyzer_executor.py

```diff
@@ -409,10 +418,17 @@ def chunker(seq: List[BaseView], size: int) -> List[List[BaseView]]:
 
 
 def emit_event(
-    analyzer_matched_subgraphs_bucket: str, s3: S3ServiceResource, event: ExecutionHit
+    analyzer_matched_subgraphs_bucket: str,
+    s3: S3ServiceResource,
+    event: ExecutionHit,
+    metadata: Metadata,
```

Comment:
I do have to wonder if we should call this like `PipelineMetadata`, but not the end of the world.

Just seems like we could eventually run into a situation where one file is juggling multiple things named Metadata (i.e. grpc)

Reply:
I think we can solve that with packaging.